spk/ffmpeg4/Makefile

```diff
@@ -1,8 +1,8 @@
-SPK_NAME = ffmpeg
+SPK_NAME = ffmpeg4
 SPK_VERS = 4.4.3
-SPK_REV = 47
+SPK_REV = 48
```

Comment:
This is not a good idea. I propose to keep `SPK_NAME = ffmpeg`
Otherwise this is a new package and you can start with SPK_REV = 1
But then all packages depending on ffmpeg (SPK_DEPENDS = "ffmpeg") will not use further updates (and we will need to keep the latest ffmpeg package in the repository).

SPK_NAME should stay ffmpeg, only DISPLAY_NAME may be changed to ffmpeg4 (or ffmpeg 4).

Reply:
The issue is more complex than that...

The github build scripts also refers to ffmpeg for it to be pre-process and keept prior to building dependent packages in order to re-use existing libraries.  That also needs fixing so a similar behavior is kept for ffmpeg5 and ffmpeg6 which adds complexity as comskip, chromaprint and tvheadend are in various stages of porting over to ffmpeg >= 5.x.

With that in mind, and based on python3 experience, I decided to rename it all to ffmpeg4 along with adjusting dependent packages.  Once all dependent packages are migrated over the new naming scheme I'll then be able to start porting them over to a newer ffmpeg.  Adding to the blur is jellyfin which requires ffmpeg 5.x for it to run smoothly (and to which there isn't any spk dependencies up until now :shrug: ).

Renaming it all to be `ffmpeg<version>` is to me the best route as they will have to be maintained in parallel for a while.  As for `SPK_REV`, I can probably switch it to `1` but on the other hand if gives the idea that `ffmpeg4` version 48 really is the update from `ffmpeg` version 47.  So I'd rather keeping it as-is.

Reply:
@th0ma7 you can name all folders ffmpeg4 only the SPK_NAME sould stay ffmpeg.
This is similar to the python2 package, where SPK_NAME is still python.